pandas/io/html.py

```diff
@@ -920,6 +921,7 @@ def _parse(flavor, io, match, attrs, encoding, displayed_only, **kwargs):
     return ret
 
 
+@deprecate_nonkeyword_arguments(version="1.4")
```

Comment:
Does `io` need to be in `allowed_args` here?

Reply:
This is not needed. If no `allowed_args` given, it defaults to the list of arguments not having the default value.